packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-group-item/gux-avatar-group-item.tsx

```diff
@@ -104,32 +105,33 @@ export class GuxAvatarGroupItem {
 
   render(): JSX.Element {
     return (
-      <button
-        type="button"
-        role="menuitem"
-        aria-label={this.name}
-        tabIndex={-1}
-        ref={el => (this.buttonElement = el)}
-        class={{
-          'gux-avatar': true,
-          [this.getAccent()]: true,
-          'gux-last-item': this.isLastItemInGroup()
-        }}
-      >
-        <slot name="image">
-          <span class="gux-avatar-initials" aria-hidden="true">
-            {generateInitials(this.name)}
-          </span>
-        </slot>
-        <gux-tooltip-beta
-          aria-hidden="true"
-          visual-only
-          placement="top"
-          ref={el => (this.tooltip = el)}
+      <Host role="menuitem">
```

Comment:
This role needs to be in the light dom, otherwise the menu won't announce the number of items in the menu